contrib/forms/xmlformgen/xslt/report.php.xslt

```diff
@@ -107,7 +107,7 @@ $lists = array(]]></xsl:text>
               $value = $dateparts[0];
             }
 
-	    echo "<td><span class='bold'>";
+	    echo "<td style='width:24%'><span class='bold'>";
```

Comment:
1. The HTML view is good. here is the image - (same for rest of the forms created with xmlformgen)

![html_view](https://cloud.githubusercontent.com/assets/1074948/25116060/fdccf9b6-2427-11e7-862d-fd39d2706343.jpg)

2. Encounter view for the rest of the xmlformgen is good...they are more compact now, except the history form in which one column has a large amount of data, it expands a bit. See image below

![encounter_view](https://cloud.githubusercontent.com/assets/1074948/25116091/31e019c2-2428-11e7-99f4-2cc4d0ebe88d.jpg)